src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -203,6 +220,17 @@ public HttpApiHandler(DurableTaskExtension config, TraceWriter traceWriter)
             return response;
         }
 
+        private static bool GetQueryParameterValue(HttpRequestMessage request, string queryParameterName)
+        {
+            if (!request.GetQueryNameValuePairs().AllKeys.Contains(queryParameterName))
+            {
+                return false;
+            }
+
+            var value = request.GetQueryNameValuePairs()[queryParameterName];
+            return !string.IsNullOrEmpty(value) && bool.Parse(value);
```

Comment:
`bool.Parse(value)` will throw if `value` is not a properly formatted boolean. I suggest the following line instead:

```csharp
return bool.TryParse(value, out bool parsedValue) && parsedValue;
```

Reply:
@cgillum Perfect! Thank you!